### DISCUSSION TITLE
How to split ppm files?

### DISCUSSION THREAD

**Discussion by dosshra** [ImageMagick/ImageMagick#2102] [Help] [answered]
I have a ppm file I need to split into 12 separated sub-images. I use the code `convert myfile.ppm -crop 25%x33.333333333333% +repage myfile_split.ppm` . I get only one file with all the sub-images. When I use this code with jpeg file I get 12 jpeg files. How can I get a file for each sub-image?
Thank you.

**Comment by snibgo** [comment, marked as answer]
Insert "+adjoin" before the output filename. Or change the output filename to include a numeric format like myfile_split_%03d.ppm.

**Reply by dosshra** [reply]
thanks, both of them did the trick
